main.go

```diff
@@ -28,14 +28,15 @@ var (
 	// CommitSHA as provided by goreleaser.
 	CommitSHA = ""
 
-	readmeNames  = []string{"README.md", "README"}
-	configFile   string
-	pager        bool
-	style        string
-	width        uint
-	showAllFiles bool
-	localOnly    bool
-	mouse        bool
+	readmeNames    = []string{"README.md", "README", "readme.md", "readme"}
```

Comment:
What about https://gitlab.com/terrakok/gitlab-client?

Reply:
oh, "Readme.md" is quite common, too, indeed